src/EditorFeatures/TestUtilities/EditAndContinue/EditAndContinueTestHelpers.cs

```diff
@@ -378,6 +378,9 @@ public static MatchingPairs ToMatchingPairs(IEnumerable<KeyValuePair<SyntaxNode,
     internal static class EditScriptTestUtils
     {
         public static void VerifyEdits<TNode>(this EditScript<TNode> actual, params string[] expected)
-            => AssertEx.Equal(expected, actual.Edits.Select(e => e.GetDebuggerDisplay()), itemSeparator: ",\r\n");
+            => AssertEx.Equal(FormatEdits(expected), FormatEdits(actual.Edits.Select(e => e.GetDebuggerDisplay())), itemSeparator: ",\r\n");
+
+        private static IEnumerable<string> FormatEdits(IEnumerable<string> edits)
```

Comment:
>FormatEdits [](http://example.com/codeflow?start=43&length=11)

AssertEx.Equal takes `itemInspector` that allows custom formating of the items in assert message.